CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+- Added attribute http.request.method to span data ([#2648](https://github.com/getsentry/sentry-dotnet/pull/2648))
```

Comment:
Why does this matter? Thinking as a user, reading this: Why would I want to update the SDK?
I believe this will show up in Sentry on the new performance view? perhaps poiniting that out will motivate ppl to upgrade

Reply:
I've updated the description for this PR to clarify what we're doing here and why. 

I don't think users will care about the this change until we phase out Transactions, to be honest. We're not providing any information that they didn't already have. This is really just an internal change to prepare us for the move to Spans. 

So I've put #skip-changelog on the description for the time being.